lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -51,6 +51,19 @@ def cancel
       keyed_fulfillment_orders
     end
 
+    def open
+      load_attributes_from_response(post(:open, {}, body.to_json))
```

Comment:
There is no `body` here, so I think this needs to be: `load_attributes_from_response(post(:open, {}, only_id))`